helpers/general.js

```diff
@@ -169,15 +169,3 @@ export function capitalize(s: string): string {
   if (typeof s !== 'string') return ''
   return s.charAt(0).toUpperCase() + s.slice(1)
 }
-
-/**
```

Comment:
did you mean to move this out of helpers/general? normally you would want to do this as a separate PR since this is a global helper file. Probably ok because nobody else seems to have used it since you added it. But why did you move it?

Reply:
unfortunately i had to move this function because my tests did not work anymore.
in this `general` there is a reference to `DataStore` and then the test always breaks, because I don't know how to mock the `DataStore`.
Of course I don't move functions which are in use by anybody else ;)

Reply:
got it. makes sense.